### PR TITLE
feat(posts): add new devfacts number 13

### PR DESCRIPTION
**Pull request by Slashgear**
## Why

Because it's fun ! 

## How to test ? 

https://deploy-preview-265--tech-m6web.netlify.app/bedrock-dev-facts-13/

### CODE REVIEW

**Review by v-gatien — APPROVED** [review]
Hell yeah :metal:

File: _posts/2021-03-02-bedrock-dev-facts-13.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+# Des branchements surprenants
+>  Ah tu test ta branche en prod ? Tu peux y aller, si tu touches pas au staging pas de soucis
+
+# Angular conventional commit
+
+Trouvé dans l'historique
+
+![screenshot d'un commit nommé 'fix(fix):fix'](/images/posts/dev-facts-13/fix.png)
+
+# Time is relativ
```
Comment (pmartin) [inline _posts/2021-03-02-bedrock-dev-facts-13.md:32]:
```suggestion
# Time is relative
```

**Review by Oliboy50 — APPROVED** [review]
clap

File: _posts/2021-03-02-bedrock-dev-facts-13.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+> Dédicace a tous ceux qui comme moi, vont passer une bonne semaine à batailler avec internet ! (au moins, j’ai assez de débit pour un mp3).
+> 320 Kbps
+
+# Apple 💰
+
+> Quand ca parle d’argent, Apple ne se rate pas généralement
+
+# Le courage
+
+> Une MEP le vendredi avec la CI KO, vous êtes mes héros :heart_eyes:
```
Comment (outadoc) [inline _posts/2021-03-02-bedrock-dev-facts-13.md:47]:
```suggestion
> Une MEP le vendredi avec la CI KO, vous êtes mes héros 😍
```

File: _posts/2021-03-02-bedrock-dev-facts-13.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+# Les petits génies
+
+> Nan mais attendez vous à plein de commentaires disant que c’est codé avec les pieds, et que Jean-Kevin, 16 ans, aurait mieux fait en 2j pendant son week-end chez sa mamie Germaine, pendant qu’il faisait une pause dans le développement du nouveau Bitcoin.
+
+# Blackhole sun
+
+En parlant de lignes supprimées.
+
+![screenshot d'une PR](/images/posts/dev-facts-13/review.png)
```
Comment (outadoc) [inline _posts/2021-03-02-bedrock-dev-facts-13.md:114]:
```suggestion
![A: Where did this go? 🤔 B: In a black hole. 😞](/images/posts/dev-facts-13/review.png)
```

File: _posts/2021-03-02-bedrock-dev-facts-13.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+# Technique de sioux
+
+> Nan mais réécrit les status 404 en 200 et ça marchera !
+
+# On peut revert une fois, mais pas quinze !
+
+![screenshot d'une PR](/images/posts/dev-facts-13/revert.png)
```
Comment (outadoc) [inline _posts/2021-03-02-bedrock-dev-facts-13.md:122]:
```suggestion
![Screenshot d'un commit dont le titre commence par: "Revert "Revert "Revert ..."](/images/posts/dev-facts-13/revert.png)
```

**Review by kennydee — APPROVED** [review]

**Review by flepretre — APPROVED** [review]

**Review by mikaelrandy — APPROVED** [review]

**Review by SofLesc — APPROVED** [review]

**Review by outadoc — APPROVED** [review]

**Pull request merged**